src/react-components/src/components/03-organisms/Hero/Hero.tsx

```diff
@@ -0,0 +1,115 @@
+// OH-30 Header with Image Right
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Image from "../../01-atoms/Images/Image/Image";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import { HeroTypes } from "./HeroTypes";
+
+export interface HeroProps {
+  /** Can be Primary, secondary, tertiary, or 50/50. */
+  heroType?: HeroTypes;
+
+  /** Required ID for tabfocus. */
+  heroId: string;
+
+  /** Required heading element. */
+  heading: JSX.Element;
+
+   /** Optional subheader that displays underneath the
+    * required heading element.
+    */
+  subHeaderText?: string;
+
+  /** Optional details area that contains location data. */
+  locationDetails?: JSX.Element;
+
+  /** Content creators can modify the foreground color
+    * when this component is used on Exhibition pages.
+    */
+  foregroundColor?: string;
```

Comment:
What happens when you pass in a non-hex color?  Can we throw a useful error?

Reply:
What happens when you pass in a color and you're not using an Exhibition hero?

Reply:
While I think it would be nice to do some type-checking on colors (e.g., you pass in `#fffffff` by mistake, we could throw an error) I would rather not error on non-hex responses. `#fff`, `#ffffff`, and `white` are all valid CSS colors.